### PR TITLE
Flow bytes pkts either/v1

### PR DESCRIPTION
**Pull request by inashivb**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/5646

### CODE REVIEW

File: tests/detect-flow-pkts-either/test.rules [hunk 1]
```diff
@@ -0,0 +1 @@
+alert ip any any -> any any (msg:"Flow has 1000 bytes in either direction";flow.pkts:either,=10;flow.bytes:either,>3000; sid:1;)
```
Comment (catenacyber) [inline tests/detect-flow-pkts-either/test.rules:1]: > Flow has 1000 bytes in either direction

does not seem to match

> flow.pkts:either,=10;flow.bytes:either,>3000

File: tests/detect-flow-pkts/test.rules [hunk 2]
```diff
@@ -1 +1 @@
-alert ip any any -> any any (msg:"Flow has 10 packets"; flow.pkts_toclient:10; flow.pkts_toserver:10; flow.bytes_toserver:<3000; flow.bytes_toclient:>8000; sid:1;)
+alert ip any any -> any any (msg:"Flow has 10 packets"; flow.pkts:toclient,=10; flow.pkts:toserver,=10; flow.bytes:toserver,<3000; flow.bytes:toclient,>8000; sid:1;)
```
Comment (catenacyber) [inline tests/detect-flow-pkts/test.rules:1]: Do you need the `=` in `=10` ?

Comment (inashivb) [reply tests/detect-flow-pkts/test.rules:1]: As far as I understood. This mode will work with or w/o the `=`. I'll remove the `=` for simplicity. Thank you. :)

File: tests/detect-flow-pkts-either/test.rules (hunk 1 above)
Comment (inashivb) [reply tests/detect-flow-pkts-either/test.rules:1]: Thank you. Will update the msg

**Pull request closed without merging**